cypher/cypher-docs/src/docs/dev/execution-plan-groups/operators.asciidoc

```diff
@@ -48,6 +48,11 @@ include::../ql/query-plan/undirected-relationship-index-seek-by-range.asciidoc[]
 // UnionNodeByLabelsScan
 include::../ql/query-plan/union-node-by-labels-scan.asciidoc[]
 
+/ DirectedAllRelationshipsScan
```

Comment:
I stole this from https://github.com/neo4j/neo4j-documentation/pull/1507, it used to be the case that this was not a necessary step but happened automatically somehow but I guess not you have to do it manually?